### PR TITLE
#4098 Fix golint issues in package influxql

### PR DESCRIPTION
**Pull request by clebs**
Fixes part of #4098

###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [ ] CHANGELOG.md updated
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: influxql/iterator_mapper.go [hunk 1]
```diff
@@ -7,18 +7,25 @@ type iteratorMapper struct {
 	auxFields []interface{}
 }
 
+// IteratorMap is an interface that provides means of retrieving an iterator by an arbitrary key.
```
Comment (jwilder) [inline influxql/iterator_mapper.go]: provides _a_ means

File: influxql/iterator_mapper.go [hunk 2]
```diff
@@ -7,18 +7,25 @@ type iteratorMapper struct {
...
+// Implementations might document how the mapping is done in each case.
 type IteratorMap interface {
 	Value(tags Tags, buf []interface{}) interface{}
 }
 
+// FieldMap is an IteratorMap that provides iterators by index
 type FieldMap int
 
+// Value provides the iterator in the Nth position in buf, beinf N the reciever's value.
```
Comment (jwilder) [inline influxql/iterator_mapper.go]: beinf?  Also "receiver's".

File: influxql/iterator_mapper.go [hunk 3]
```diff
@@ -7,18 +7,25 @@ type iteratorMapper struct {
...
 func (i FieldMap) Value(tags Tags, buf []interface{}) interface{} { return buf[i] }
 
+// TagMap is an IteratorMap that provides iterators by tag
 type TagMap string
 
+// Value provides the iterator mapped to the reciever's tag
```
Comment (jwilder) [inline influxql/iterator_mapper.go]: receiver's

File: influxql/iterator_mapper.go [hunk 4]
```diff
@@ -7,18 +7,25 @@ type iteratorMapper struct {
...
 func (s TagMap) Value(tags Tags, buf []interface{}) interface{} { return tags.Value(string(s)) }
 
+// NewIteratorMapper creates an empty iteratorMapper ready for use
```
Comment (jwilder) [inline influxql/iterator_mapper.go]: Missing period at end.

File: influxql/iterator_mapper.go (hunk 3 above)
Comment (jwilder) [inline influxql/iterator_mapper.go]: Missing period at end.

File: influxql/iterator_mapper.go [hunk 5]
```diff
@@ -7,18 +7,25 @@ type iteratorMapper struct {
...
 	Value(tags Tags, buf []interface{}) interface{}
 }
 
+// FieldMap is an IteratorMap that provides iterators by index
 type FieldMap int
 
+// Value provides the iterator in the Nth position in buf, beinf N the reciever's value.
 func (i FieldMap) Value(tags Tags, buf []interface{}) interface{} { return buf[i] }
 
+// TagMap is an IteratorMap that provides iterators by tag
```
Comment (jwilder) [inline influxql/iterator_mapper.go]: Missing period at end.

**Comment by clebs** [comment]
Hi @jwilder, thanks for the feedback!
I have corrected all typos.
Please let me know if you have any other comments.

**Comment by jsternberg** [comment]
The `influxql` package got broken out and moved into [influxdata/influxql](https://github.com/influxdata/influxql) so I'm going to close this.

**Pull request closed without merging**